modin/core/storage_formats/pandas/groupby.py

```diff
@@ -245,3 +245,260 @@ def mean_reduce(dfgb, **kwargs):
     "skew": GroupbyReduceImpl._build_skew_impl(),
     "sum": ("sum", "sum", lambda grp, *args, **kwargs: grp.sum(*args, **kwargs)),
 }
+
+
+class PivotTableImpl:
```

Comment:
`.pivot_table()` is literally a groupby + fancy post-processing, so decided to put it into `groupby.py`